lib/test/unit/test-suite-runner.rb

```diff
@@ -10,6 +10,7 @@ module Test
   module Unit
     class TestSuiteRunner
       @default = self
+      @n_workers = 5
```

Comment:
Could you use `Etc.nprocessors` as the default value?

```suggestion
      @n_workers = Etc.nprocessors
```

Reply:
Sure!